contrib/pyzfs/libzfs_core/_constants.py

```diff
@@ -106,7 +106,7 @@ def enum(*sequential, **named):
 )
 # compat before we used the enum helper for these values
 ZFS_ERR_CHECKPOINT_EXISTS = zfs_errno.ZFS_ERR_CHECKPOINT_EXISTS
-assert(ZFS_ERR_CHECKPOINT_EXISTS == 1024)
+assert (ZFS_ERR_CHECKPOINT_EXISTS == 1024)
```

Comment:
Can be dropped was fixed in master by 51946eda70810d346c12b8f0f97cf95441e0bf66

Reply:
Sure, will do.

Reply:
I have removed this change.